packages/@sanity/form-builder/src/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -55,7 +91,7 @@ export type PortableTextInputProps = {
   value: PortableTextBlock[] | undefined
 }
 
-const PortableTextInputWithRef = React.forwardRef(function PortableTextInput(
+const PortableTextInputController = React.forwardRef(function PortableTextInputController(
   props: Omit<PortableTextInputProps, 'level'>,
   ref: React.RefObject<PortableTextEditor>
```

Comment:
I believe the correct typing here is:
```suggestion
  ref: React.ForwardedRef<PortableTextEditor>
```

Reply:
I'm getting error that it doesn't have `.current` when trying to use that one.